=

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import Image from 'next/image'
+// MUI //
+import Drawer from '@mui/material/Drawer'
+import { Box } from '@mui/system'
+import Collapse from '@mui/material/Collapse'
+// other mui dependencies
+import ExpandLess from '@mui/icons-material/ExpandLess'
+import ExpandMore from '@mui/icons-material/ExpandMore'
+
+// icons and text
+import AccessTimeIcon from '@mui/icons-material/AccessTime'
+import StarIcon from '@mui/icons-material/Star'
+import SettingsIcon from '@mui/icons-material/Settings'
+
+export default function NavigationDrawer() {
+  // Params for the Drawer
+  const drawerWidth = 280
+  // Logo definition
+  const logo = (
+    <Image
+      src="/../public/images/CCAHT-Logo.png"
+      alt="CCAHT-logo"
+      height="70"
+      width="239"
+    />
+  )
+  // state handlers for expandable lists in drawer
+  const [open, setOpen] = React.useState(true)
+  const handleClick = () => {
+    setOpen(!open)
+  }
+  // Drawer setup
+  // TODO ICONS
+  return (
+    <Box sx={{ display: 'flex' }}>
+      <Drawer
+        variant="permanent"
+        sx={{
+          width: drawerWidth,
+        }}
+      >
+        <Box>
```

Comment:
this needs to have prop `sx={{ width: drawerWidth }}` so that the drawer is sized correct, its weird.

Reply:
Are you saying the <Box> needs the drawerwidth?

Reply:
yea, when i was playing with it i noticed the size of the drawer element correct, but the displayed drawer width was not. when i checked out the example in the docs it showed that the box inside the drawer had the width set.